env/Dockerfile

```diff
@@ -84,30 +92,6 @@ RUN cd cmake-3.20.0 \
 RUN wget http://ftp.pl.debian.org/debian/pool/main/n/ninja-build/ninja-build_1.10.1-1_amd64.deb \
     && dpkg -i ninja-build_1.10.1-1_amd64.deb
 
-# Install gcc 9.3.0
-RUN apt remove gcc -y
-
-RUN wget http://ftp.pl.debian.org/debian/pool/main/g/gcc-9/gcc-9-base_9.3.0-22_amd64.deb \
-&& dpkg -i gcc-9-base_9.3.0-22_amd64.deb
-
-RUN wget http://ftp.pl.debian.org/debian/pool/main/g/gcc-9/cpp-9_9.3.0-22_amd64.deb \
-&& dpkg -i cpp-9_9.3.0-22_amd64.deb
-
-RUN wget http://ftp.pl.debian.org/debian/pool/main/g/gcc-10/gcc-10-base_10.2.1-6_amd64.deb \
-&& dpkg -i gcc-10-base_10.2.1-6_amd64.deb
-
-RUN wget http://ftp.pl.debian.org/debian/pool/main/g/gcc-10/libtsan0_10.2.1-6_amd64.deb \
-&& dpkg -i libtsan0_10.2.1-6_amd64.deb
-
-RUN wget http://ftp.pl.debian.org/debian/pool/main/g/gcc-9/libasan5_9.3.0-22_amd64.deb \
-&& dpkg -i libasan5_9.3.0-22_amd64.deb
-
-RUN wget http://ftp.pl.debian.org/debian/pool/main/g/gcc-9/libgcc-9-dev_9.3.0-22_amd64.deb \
-&& dpkg -i libgcc-9-dev_9.3.0-22_amd64.deb
-
-RUN wget http://ftp.pl.debian.org/debian/pool/main/g/gcc-9/gcc-9_9.3.0-22_amd64.deb \
-&& dpkg -i gcc-9_9.3.0-22_amd64.deb
-
```

Comment:
Why are you removing dependancies needed to build the QEMU image? The goal was to add support for building the CROSSCON HV for Raspberry PI but we still want to be able to build image for QEMU. Maybe you could try docker compose if conflicting gcc are creating problems

Reply:
@tym2k1 i am not removing dependencies - in this state they were simply unusable. I understand this was done to have the exact same version as in the readme, but this just left us with an absolutely unusable container which couldnt build buildroot because the binaries werent in the place that buildroot expected them to be. I tried to work around this by creating symlinks and copying the binaries but to no avail. It was simpler to just install gcc in the most basic way possible - it doesnt seem to matter which specific gcc version we have. The current version of the container should be able to build both the QEMU demo (riscv included) and the rpi4-ws demo.

Reply:
If the `gcc` version mentioned in the CROSSCON HV README is unneccessary I think we should file an issue there. We're working on this container so that every CROSSCON project partner has a standarized dev environment. What do you mean by unusable? I might've missed this particular problem you encountered. We can continue this discussion here or internally on what were the exact problems and how we can fix them

Reply:
Closing this thread as it has been discussed internally and also mentioned here https://github.com/crosscon/CROSSCON-Hypervisor-and-TEE-Isolation-Demos/issues/6